OxfordBodleian/Juel-Jensen/BDLaethb3.xml

```diff
@@ -889,26 +889,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                <persName ref="PRS13900WaldaD" role="scribe"/> is mentioned in black ink in the lower margin of <locus target="#117r #138v"/>.
-                                The two different names mentioned suggest at least two scribes were involved in the copying,
-                                but no discontinuity in the hand can be observed throughout the manuscript. 
-                                -->
-                                <date notBefore="1730" notAfter="1755"/>
-                                <desc>Regular handwriting, trained hand, <foreign xml:lang="gez">gʷǝlḥ</foreign> script. Big and evenly spaced characters. The two different names indicated in the bottom margins
-                                    of <locus target="#117r #138v"/> and <locus target="#148r"/> indicate that at least two scribes were involved in the copying,
-                                but no differences in the hand can be observed throughout the manuscript. The script of the introductions to the Gospels is smaller than that of the main text.</desc>
-                                <!-- 
-                                Regular handwriting, trained hand, <hi rendition="simple:italic">gʷǝlḥ</hi>-script. Characters are big and evenly spaced.
-                                The script of the Introductions to the Gospels is smaller than that of the main text.
-                                -->
+                                <date notBefore="1730" notAfter="1755"/> 
                                 <seg type="ink">Black, red.</seg>
-                                <seg type="rubrication">Single lines and groups of lines at the beginning of sections; single lines alternating with black lines
-                                    at the beginning of each Gospel; chapter indications; holy names; names mentioned in the supplications;
-                                    numerals and elements of numerals and punctuation signs.</seg>
+                                <desc>                                    
+                                    Regular handwriting, trained hand, <hi rendition="simple:italic">gʷǝlḥ</hi>-script. Characters are big and evenly spaced.
+                                    <persName ref="PRS13901Zaparaqlitos" role="scribe"/> is mentioned in red ink in the lower margin of <locus target="#148r"/>.
+                                    <persName ref="PRS13900WaldaD" role="scribe"/> is mentioned in black ink in the lower margin of <locus target="#117r #138v"/>.
+                                    The two different names mentioned suggest at least two scribes were involved in the copying,
+                                    but no discontinuity in the hand can be observed throughout the manuscript. 
+                                    The script of the Introductions to the Gospels is smaller than that of the main text.
```

Comment:
Should we mark up "Introduction to the Gospels" with a reference to the text part? Or is it unnecessary?